assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -284,6 +284,7 @@ che.openshift.project=eclipse-che
 che.openshift.serviceaccountname=cheserviceaccount
 che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
+che.openshift.persistent.volume.claim=claim-che-data
```

Comment:
Since this property is about PVC for workspaces I would rather call it: `che.openshift.workspaces.pvc.name`. What do yo think?